added load_stac example notebook

VictorVerhaert opened the pull request:
issue #30

VictorVerhaert commented:
feedback from @HansVRP : add plots to show the results

Pratichhya commented on python/LoadStac/load-stac-item-example.ipynb:
```diff
@@ -0,0 +1,253 @@
+    "collection = collection.process(\"mask_scl_dilation\", scl_band_name=\"SCL\", data=collection)\n",
+    "collection = collection.ndvi(red=\"B04\",nir=\"B08\", target_band=\"NDVI-band\").filter_bands(\"NDVI-band\")\n",
+    "collection = collection.reduce_temporal(\"mean\")\n",
+    "collection.download(\"ndvi-file.tiff\", format=\"GTiff\")"
```
In synchronous, specifying the output format might not be needed since you already define the output with the format.

Pratichhya reviewed (commented):
Please ignore if any comments are irrelevant 😃

VictorVerhaert commented on python/LoadStac/load-stac-item-example.ipynb:
```diff
@@ -0,0 +1,253 @@
+    "    max_cloud_cover=95\n",
+    ")\n",
+    "collection = collection.process(\"mask_scl_dilation\", scl_band_name=\"SCL\", data=collection)\n",
+    "collection = collection.ndvi(red=\"B04\",nir=\"B08\", target_band=\"NDVI-band\").filter_bands(\"NDVI-band\")\n",
```
I specified it because I did it wrong the first time...

HansVRP commented:
I wanted to drop a quick note to express my appreciation for including the images in the notebook. It really enhances the rigor of our documentation!

I also had a thought about the STAC builder. What if we consider including two layers instead of just one? My thinking is that it might be more common for users to include multiple bands in their analysis, so having an example that reflects this could be really beneficial.

Let me know what you think about this idea, and if you believe it could add value to the notebook.

jdries commented:
Very nice example!
* there's a typo in `Getting an example GTiff file fron openEO`
* you can replace temporal extent with something like '2022-06', showcasing this nifty feature: https://open-eo.github.io/openeo-python-client/data_access.html#single-string-temporal-extents


Maybe one extra thing to mention: here we construct the stac metadata ourselves, for the sake of the example, but it is very important to highlight that metadata generated by openEO can also be used directly in load_stac, so there's no need to generate your own metadata when working with openEO results.

VictorVerhaert commented:
I have fixed the typo and changed the temporal range. 

@HansVRP for now I'd leave out the second band to keep it simple.

@jdries The load stac of a previous job isn't fully supported yet I think, so I suggest creating a seperate example notebook for this. (and then reference it later in this notebook)

Let me know what you think!

HansVRP commented:
@jdries would you prefer the metadata example, or do we merge as is?

jdries commented:
merge as is!
